osmaxx/production_example.yml

```diff
@@ -0,0 +1,37 @@
+version: '2'
+services:
+  osmaxx-proxy:
+    environment:
+      # comma separated list, no brackets, e.g. localhost,dev.myhost.com
+      - VIRTUAL_HOST=<my.domain.org>
```

Comment:
As long as there are placeholders that aren't valid values, this isn't really an example, is it?